deploy/registry/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM registry:2.6.2
-RUN apk add --no-cache --update curl
+FROM registry:2.7.1
```

Comment:
Can you pin only to 2.7 please, want make sure we get those bug fixes.

Reply:
I would prefer to pin to a specific revision, but happy for someone else to fettle and tweak to their heart's content once merged.

Reply:
Thats fine with me.